modules/distribution/carbon-home/conf/manager/deployment.yaml

```diff
@@ -290,4 +290,7 @@ deployment.config:
   datasource: SP_MGT_DB           # define a mysql datasource in datasources and refer it from here.
   minResourceCount: 1
   bootstrapURLs: localhost:9092   # kafka urls
-  zooKeeperURLs: localhost:2181   # zookeeper urls
+  zooKeeperConfig:
```

Comment:
Don't we need to change bean classes to comply to this change?

Reply:
Changes done in carbon-analytics#1233

Reply:
Changes done in wso2/carbon-analytics#1233